### PR TITLE
Add in tabs component

### PR DESCRIPTION
**Pull request by bradcypert**
Screen Shot 2020-11-03 at 3 03 36 PM

Add in a React component for managing tabs :)

### CODE REVIEW

File: src/js/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/**
+ * @name Avatar
+ *
+ * @description
+ *  A simple component used to display an avatar for a user or organization.
```
Comment (saralohr) [inline src/js/components/Tabs/Tabs.tsx:5]: Pulling in some Avatar documentation.

**Review by saralohr — APPROVED** [review]
Looks good 👍

File: src/js/components/Tabs/Tabs.tsx (hunk 1 above)
Comment (bradcypert) [reply src/js/components/Tabs/Tabs.tsx:5]: Whoops, ill fix that xD

File: src/js/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/**
+ * @name Avatar
```
Comment (dbhaasis) [inline src/js/components/Tabs/Tabs.tsx:2]: Wrong name?

**Review by dbhaasis — APPROVED** [review]
Looks nice and clean

**Comment by bradcypert** [comment]
I'm going to merge this into the avatar branch and we can release the two new features together :D

**Pull request merged**